apps/studio/components/interfaces/DiskManagement/DiskManagementRestartRequiredSection.tsx

```diff
@@ -0,0 +1,42 @@
+import { AnimatePresence, motion } from 'framer-motion'
+import { RotateCcw } from 'lucide-react'
+import React from 'react'
+import { DialogSection, WarningIcon } from 'ui'
+
+interface Props {
+  visible: boolean
+  title: string
+  description: string
+}
+
+export const DiskMangementRestartRequiredSection: React.FC<Props> = ({
```

Comment:
we've moved away from using FC in particular if the component doesn't need children

naming consistency: DiskManagementRestartRequiredSectionProps
so just `export const DiskMangementRestartRequiredSection = ({ ...}: DiskManagementRestartRequiredSectionProps) => { ... }`